lib/publiccloud/instance.pm

```diff
@@ -616,12 +616,25 @@ sub measure_boottime() {
     $ret->{kernel_release} = $instance->run_ssh_command(cmd => 'uname -r', proceed_on_failure => 1);
     $ret->{kernel_version} = $instance->run_ssh_command(cmd => 'uname -v', proceed_on_failure => 1);
 
-    # Do logging to openqa UI
     $Data::Dumper::Sortkeys = 1;
+    my $dir = "/var/log";
+    my @logs;
+    my @missing_logs;
+
+    # Select existing files
+    (-e $dir . '/' . $_) ? push @logs, $_ : push @missing_logs, $_
```

Comment:
Doesn't `-e` test the file on the local filesystem, i.e. on the openQA worker? :thinking:

Reply:
checking...

Reply:
I think you are right Felix . I got confused that we have part of tests where we switching to ssh console and run transparently calls to `assert_script_run` but seems like here is not the case . @m-dati  you need to switch to using `$instance-run_ssh_command(cmd => 'test -e' . $_)` here ...

Reply:
yes I just fixed it :slightly_smiling_face: